examples/test-hello-image.py

```diff
@@ -1,12 +0,0 @@
-from ee_plugin import Map
-import ee
-
-image = ee.Image.pixelLonLat() \
-    .add([180, 90]).divide([360, 180])
-
-# image = image.multiply(50).sin()
-
-Map.addLayer(image, {}, 'coords', True)
-
-
-
```

Comment:
Why is this file deleted?

Reply:
I didn't delete, I renamed the example files and added one

Reply:
Ah, I did not notice this. I've merged everything into master.